cachi2/core/package_managers/rpm/main.py

```diff
@@ -359,11 +360,18 @@ def raise_exception(message: str) -> None:
                 raise_exception(f"Unmatched checksum of '{file_path}' != '{digest}'")
 
 
+def _is_rpm_file(file_path: Path):
+    """Check if it's a rpm file."""
+    return file_path.suffix == ".rpm"
+
+
```

Comment:
I will politely disagree. A simple boolean comparison tells a maintainer how one achieves some goal, a well-named function tells what the goal is. Immediately knowing a goal reduces mental load and makes code more straightforward. Here the goal is to check if some object refers to an rpm file and it happens that it boils down to a simple comparison. If at some point in time this becomes inadequate to the goal then it could be easily adjusted.

The overhead of a function call in this specific case (in most other cases too) is negligible, this is not a tight loop, the code above deals with downloading data which will always dominate by several orders of magnitude. Also this is Python after all, it is usually valued for readability, not for lack of overhead. I would still suggest keeping this function.

Reply:
What you argue about makes sense...when there are multiple places where such a check could be employed, there's literally a single spot where this is used in a very distinct context, so I'm still not convinced.

Reply:
It's not a blocker for me though.